lib/tjbot.js

```diff
@@ -30,7 +30,6 @@ const spawn = require('child_process').spawn;
 const winston = require('winston');
 
 // hardware modules
-const APlay = require('aplay');
 const Mic = require('mic');
 const Raspistill = require('node-raspistill').Raspistill;
```

Comment:
Quick question ...
`const Raspistill = require('node-raspistill').Raspistill;`
Would this be a problem for people who dont have a camera? Value in lazy loading this to only when camera is indicated as a hardware flag?